README.md

```diff
@@ -26,7 +26,7 @@ We are also in the process of preparing a detailed paper specifically about this
 ∞RETIS can be installed via [pip](https://pypi.org/project/infretis/):
 
 ```bash
-pip install infretis
+pip install git+https://github.com/infretis/infretis.git@main
```

Comment:
just for sanity it is probably better to do 
```suggestion
python -m pip install git+https://github.com/infretis/infretis.git@main
```
to make sure it is installed into the correct python env (this used to be a hassle in the early days of conda/pyenv)